roles/kibana/proxy/tasks/main.yml

```diff
@@ -5,13 +5,27 @@
     dest: '{{ kibana_httpd_conf }}'
   notify: restart httpd
 
+- name: check if kibana_proxy_user is contained in kibana_proxy_htpasswd
+  shell: '/usr/bin/cat {{kibana_proxy_htpasswd}} |\
```

Comment:
This is failing the travis checks with `ANSIBLE0012`, because this tasks runs a command every time.  You can suppress lint checks on the task by adding the same tag you see on the following task:

```
tags:
    - skip_ansible_lint  # ignore ANSIBLE0012
```

Note that the comment is informative only (it has no impact on ansible-lint; this tag suppresses all checks for the task).